src/js/components/orgs/cards/orgActions.tsx

```diff
@@ -138,23 +140,76 @@ const OrgActions = ({
     const needsReview =
       task.has_unmerged_commits && task.pr_is_open && !task.review_valid;
     let isActive = false;
+    let popover = null;
+
     switch (type) {
       case ORG_TYPES.DEV:
         isActive = hasReviewRejected || !task.has_unmerged_commits;
+        popover = (
+          <TourPopover
+            align="right"
+            heading={i18n.t('Create a Dev Org')}
+            body={
+              <Trans i18nKey="tourTaskCreateDevOrg">
+                A Dev Org is a temporary Salesforce org where you can make
+                changes that you would like to contribute to the Project. To
+                create an Org, make sure you are connected to a Salesforce
+                account with Dev Hub enabled. Use the drop down menu to delete
+                the Dev Org when you no longer need it.
+              </Trans>
+            }
+          />
+        );
+
         break;
       case ORG_TYPES.QA:
         isActive = needsReview;
+        popover = (
+          <TourPopover
+            align="right"
+            heading={i18n.t('Create a Test Org')}
+            body={
+              <Trans i18nKey="tourTaskCreateTestOrg">
+                A Test Org is a temporary Salesforce org where you can view the
+                changes the Developer retrieved. To create an Org, make sure you
+                are connected to a Salesforce account with Dev Hub enabled. Read
+                the Developer’s Commit History to see what changes they made.
+                Use the drop down menu to delete the Test Org when you no longer
+                need it.
+              </Trans>
+            }
+          />
+        );
         break;
     }
     return (
       <>
-        {submitReviewBtn}
+        {submitReviewBtn ? (
+          <span className="slds-is-relative">
+            {submitReviewBtn}
+            <TourPopover
+              align="right"
+              heading={i18n.t('Submit a review')}
+              body={
+                <Trans i18nKey="tourTaskSubmitReview">
+                  When you’re finished viewing and testing all the changes, come
+                  back to Metecho to leave your review. You will have the option
+                  to approve the work or request changes. Clearly describe any
+                  further changes you’d like the Developer to make.
+                </Trans>
+              }
+            />
+          </span>
+        ) : null}
```

Comment:
This is pretty minor, but it looks like when both of these buttons are visible, the "Create Org" btn appears on top of the "Submit Review" popover icon:

![Screen Shot 2021-05-24 at 12 05 13 PM](https://user-images.githubusercontent.com/552316/119375323-ad0fcc80-bc88-11eb-97fc-41272d9b35e7.png)

Reply:
I'll work on adding this space when these appear together.

Reply:
Ok -- if it's not simple then we can ignore it; it's hardly noticeable.

Reply:
Fixing the z-index helped here. re-positioned the popovers a bit too. 
![image](https://user-images.githubusercontent.com/41588129/119496704-0e589e00-bd64-11eb-8cdf-701aa44885e6.png)